hm_pyhelper/diagnostics/diagnostics_report.py

```diff
@@ -50,8 +46,25 @@ def set_passed(self, passed):
     def append_error(self, key):
         self.__getitem__(DIAGNOSTICS_ERRORS_KEY).append(key)
 
-    def get_errors(self):
-        return self[DIAGNOSTICS_ERRORS_KEY]
+    def has_errors(self, keys_to_check: Union[tuple, list, set] = None) -> set:
+        """
+        Return list of keys that have errors (don't have valid values).
+
+        Args:
+            keys_to_check (Union[tuple, list, set, None]): If not None then
```

Comment:
Would be helpful to add `If None, return all errors.`

Reply:
Yeah, should have added that. Done now.